tests/ps_test.go

```diff
@@ -147,7 +147,9 @@ var _ = Describe("deis ps", func() {
 					afterProcs := scrapeProcs(app.Name, procsListing)
 
 					// compare the before and after sets of process names
-					Expect(len(afterProcs)).To(Equal(scaleTo))
+					if restart != "one" { // current workaround for an issue in deis/controller
```

Comment:
Is there an issue to track that we want to restore this test check in the future? And I'd add a `TODO:` or `HACK:` comment prefix to underline that idea.

Also, e2e has passed without this change, so I wonder if it is truly needed now.